Sandbox/src/Game/Interface/GearInterface.cpp

```diff
@@ -1,17 +1,83 @@
 #include "GearInterface.h"
 
 GearInterface::GearInterface(Gear& gear)
-	: IMenuTab("Graphics/panel_brown.png", 32, false, false), mGear(gear)
+	: IMenuTab("Graphics/panel_gear.png", 32, false, false), mGear(gear)
 {
-	for (int y = 0; y < 5; y++)
+	InitializeGearPositions();
+}
+
+void GearInterface::InitializeGearPositions()
+{
+	mGearPositions = {
+	{ Gear::EquipmentSlot::head, {COL_THREE_X, ROW_ONE_Y} },
+	{ Gear::EquipmentSlot::cape, {COL_TWO_X, ROW_TWO_Y} },
+	{ Gear::EquipmentSlot::neck, {COL_THREE_X, ROW_TWO_Y} },
+	{ Gear::EquipmentSlot::weapon, {COL_ONE_X, ROW_THREE_Y} },
+	{ Gear::EquipmentSlot::chest, {COL_THREE_X, ROW_THREE_Y} },
+	{ Gear::EquipmentSlot::shield, {COL_FIVE_X, ROW_THREE_Y} },
+	{ Gear::EquipmentSlot::legs, {COL_THREE_X, ROW_FOUR_Y} },
+	{ Gear::EquipmentSlot::gloves, {COL_ONE_X, ROW_FIVE_Y} },
+	{ Gear::EquipmentSlot::boots, {COL_THREE_X, ROW_FIVE_Y} },
+	{ Gear::EquipmentSlot::ring, {COL_FIVE_X, ROW_FIVE_Y} },
+	{ Gear::EquipmentSlot::ammo, {COL_FOUR_X, ROW_TWO_Y} },
+	};
+
+	mGearDefaultSprites = {
+	{ Gear::EquipmentSlot::head, Square::Sprite("Graphics/head.png") },
+	{ Gear::EquipmentSlot::cape, Square::Sprite("Graphics/cape.png") },
+	{ Gear::EquipmentSlot::neck, Square::Sprite("Graphics/neck.png") },
+	{ Gear::EquipmentSlot::weapon, Square::Sprite("Graphics/weapon.png") },
+	{ Gear::EquipmentSlot::chest, Square::Sprite("Graphics/chest.png") },
+	{ Gear::EquipmentSlot::shield, Square::Sprite("Graphics/shield.png") },
+	{ Gear::EquipmentSlot::legs, Square::Sprite("Graphics/legs.png") },
+	{ Gear::EquipmentSlot::gloves, Square::Sprite("Graphics/gloves.png") },
+	{ Gear::EquipmentSlot::boots, Square::Sprite("Graphics/boots.png") },
+	{ Gear::EquipmentSlot::ring, Square::Sprite("Graphics/ring.png") },
+	{ Gear::EquipmentSlot::ammo, Square::Sprite("Graphics/ammo.png") },
+	};
+}
+
+void GearInterface::CustomRender()
```

Comment:
We don't need a new render method. We can easily add a line or two modifying the base class Render method.